rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml

```diff
@@ -41,10 +62,8 @@ note = """## Triage and analysis
 ### Response and remediation
 
 - Initiate the incident response process based on the outcome of the triage.
-- If the host is a domain controller (DC):
-  - Activate your incident response plan for total Active Directory compromise.
-  - Review the privileges assigned to users that can access the DCs, to ensure that the least privilege principle is being followed and to reduce the attack surface.
-- Isolate the involved hosts to prevent further post-compromise behavior.
+- Isolate the source host to prevent further post-compromise behavior.
+- If the asset is exposed to the internet with RDP or other remote services available, take the necessary measures to restrict access to the asset by not exposing it preferentially. If not possible, limit the access via the firewall to only the needed IP addresses. Also, ensure the system uses robust authentication mechanisms and is patched regularly.
```

Comment:
Same as above, "by not exposing it preferentially" is hard to follow. Remove or rephrase?